tutorials/director/README.md

```diff
@@ -1,16 +1,22 @@
 This folder provides a minimal Dockerfile and k8s resource definition to build a sample director for Open Match tutorials.
 
+Run the command below to define a variable for your personal container registry in your current terminal session
+```
+# Example - replace open-match-public-images with your personal registry ID
+YOUR_PERSONAL_REGISTRY=gcr.io/open-match-public-images
```

Comment:
nit - rename to 

`# Specify your Registry URL here.
`
REGISTRY=[YOUR_REGISTRY_URL]